docs/Developers/Newcomers_Guide.rst

```diff
@@ -30,10 +22,8 @@ following tasks, you will become a developer at coala:
   - review at least a ``difficulty/low`` or higher Pull Request
   - do at least a `meta-review <Meta-review.html>`_ or receive a meta-review
 
-.. note::
-
-    After you have solved a ``difficulty/newcomer`` issue, please don't
-    take up any more. Instead, move on to more difficult issues.
+Note: After you have solved a ``difficulty/newcomer`` issue, please don't
```

Comment:
@AditiKatiyar  there is no need to change these lines ,issue is about adding documentation for verifying whether issue exist or not

Reply:
i didn't intend to change any of the existing code. i don't know how it happened. i am fixing all of this now. right now i am having issues with versions of dependencies because of which 'coala' is failing to run.
Also, i would like to know whether i should add the step at the end or in the middle of the documentation somewhere?

Reply:
Like AkshJain said no need to change anything here. re-add like it was exactly